### ISSUE TITLE
Multiple sliders start together even if they have different entities

### ISSUE THREAD

**Issue by tehnic1service** [rianadon/timer-bar-card#112]
<img width="533" alt="image" src="https://github.com/rianadon/timer-bar-card/assets/122440223/ae4db815-3bac-418a-8ca9-5542f2bf0c92">

I have 2 cards, each having diferent entity, but when I start one, both timers start. This is only a visual problem, because only the entity I trigger is actually switched.

**First card configuration:**

```yaml
type: custom:timer-bar-card
entity: switch.sprinklere_gazon_spate
name: Gazon spate
duration:
  fixed: '00:15:00'
invert: true
bar_direction: rtl
bar_width: 60%
mushroom:
  layout: horizontal
  color: green
hold_action:
  action: toggle
```
**Second card configuration:**

```yaml
type: custom:timer-bar-card
entity: switch.sprinklere_gazon_fata
name: Gazon fata
duration:
  fixed: '00:15:00'
invert: true
bar_direction: rtl
bar_width: 60%
mushroom:
  layout: horizontal
  color: green
hold_action:
  action: toggle
```

**Debug information**

GIVEN I start the first timer
THEN both timers countdown start:

<img width="515" alt="image" src="https://github.com/rianadon/timer-bar-card/assets/122440223/fc9c8067-c613-4d5b-8ef6-9dc33c4cb6c8">

GIVEN I start the first timer
WHEN I stop the first timer
THEN both timers stop:

<img width="539" alt="image" src="https://github.com/rianadon/timer-bar-card/assets/122440223/8dc2b76c-3839-47f6-8320-dc4503ca2b44">

GIVEN I start the first timer
WHEN I stop the second timer
THEN the first timer continue running:

<img width="551" alt="image" src="https://github.com/rianadon/timer-bar-card/assets/122440223/c35cd57f-37fa-4f47-8642-45471df58e44">

I'm not sure if I'm doing it right because I just started with Home Assistant and custom cards. If I can send additional data, please let me know. 
Thank you!

**Comment by rianadon** [comment]
Hi! Welcome to Home Assistant ✨. There really is a lot to learn. And thank you for all the screenshots. This is super helpful.

The `State` field shown in the debugging panel is taken directly from Home Assistant, so it almost appears as if whatever integration is connecting these switches is reporting the incorrect state.

Would you mind trying to turn on/off the switches using the built-in Home Assistant controls in the same set of sequences you posted above to see if they behave differently?

**Comment by tehnic1service** [comment]
I did some more tests, and you are right. The entity I call, is a group of 3 entities: The pump, the power supply of the valves and the specific valve. So both  timers have the pump and the power supply in the group. This is why both look active when I trigger one of them.

I made another timer with only the sprinkler valve and it worked as expected. Only that timer starts when I trigger it. 
<img width="499" alt="image" src="https://github.com/rianadon/timer-bar-card/assets/122440223/f18ba555-6982-4c39-9ea8-993ffcd072d2">

Probably I'll add an automation, to trigger the pump and the power supply when a sprinkler valve is triggered.

Thank you for your help and for the great timer you made! 🍺
